apps/contract-sync-job/src/db.ts

```diff
@@ -74,9 +74,10 @@ export async function getContribution(data: { tokenId: number }) {
     ).result;
 
     if (contrs.length != 0) return contrs[0].id;
-    throw new Error('Contribution must exist for this attestation!');
+    throw new Error('Contribution is missing.');
```

Comment:
This error message should probably be more clear. I think the original is more specific then this change.